backend/src/main/java/com/nakaligoba/backend/controller/ProjectController.java

```diff
@@ -0,0 +1,31 @@
+package com.nakaligoba.backend.controller;
+
+import com.nakaligoba.backend.service.ProjectService;
+import com.nakaligoba.backend.service.ProjectService.*;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/v1/projects")
+public class ProjectController {
+
+    private final ProjectService projectService;
+
+
+    @PostMapping
+    public ResponseEntity<ProjectCreateResponse> createProject(@Valid @RequestBody ProjectCreateRequest request) {
+        ProjectCreateResponse response = projectService.create(request);
+        return ResponseEntity.ok(response);
+    }
+
+    @GetMapping
+    public ResponseEntity<?> getAllProjects() {
```

Comment:
어떤 응답을 넘기는지 타입 변수로 넣는게 좋을 것 같습니다
```suggestion
    public ResponseEntity<ProjectListResponse> getAllProjects() {
```